### PR TITLE
Event Log Fixed - MoodNft

### PR DESCRIPTION
**Pull request by theirrationalone**
1. Modified: /src/MoodNft.sol
2. Updated: /test/MoodNftTtest.sol

# Tests Passed without any issue 👇
Screenshot (139)

### CODE REVIEW

**Comment by PatrickAlphaC** [comment]
Thanks for this!

So a BIG thing you can see here is that the formatting changed a lot, making this PR difficult to review. Can you remove all the formatting changes you made?

**Review by theirrationalone — COMMENTED** [review]
Heyy @PatrickAlphaC !, Formatting Fixed. Done! 😅

;`)

File: src/MoodNft.sol [hunk 1]
```diff
@@ -54,9 +54,10 @@ contract MoodNft is ERC721, Ownable {
 
     function mintNft() public {
         // how would you require payment for this NFT?
-        _safeMint(msg.sender, s_tokenCounter);
+        uint256 _tokenCounter = s_tokenCounter;
```
Comment (PatrickAlphaC) [inline src/MoodNft.sol:57]:
```suggestion
        uint256 tokenCounter = s_tokenCounter;
```

File: src/MoodNft.sol [hunk 2]
```diff
@@ -54,9 +54,10 @@ contract MoodNft is ERC721, Ownable {
...
+        _safeMint(msg.sender, _tokenCounter);
         s_tokenCounter = s_tokenCounter + 1;
-        emit CreatedNFT(s_tokenCounter);
+        emit CreatedNFT(_tokenCounter);
```
Comment (PatrickAlphaC) [inline src/MoodNft.sol:60]:
```suggestion
        emit CreatedNFT(tokenCounter);
```

File: src/MoodNft.sol [hunk 3]
```diff
@@ -54,9 +54,10 @@ contract MoodNft is ERC721, Ownable {
 
     function mintNft() public {
         // how would you require payment for this NFT?
-        _safeMint(msg.sender, s_tokenCounter);
+        uint256 _tokenCounter = s_tokenCounter;
+        _safeMint(msg.sender, _tokenCounter);
```
Comment (PatrickAlphaC) [inline src/MoodNft.sol:58]:
```suggestion
        _safeMint(msg.sender, tokenCounter);
```

**Review by PatrickAlphaC — COMMENTED** [review]
Can we use this syntax instead? I don't love the leading underscore

**Comment by PatrickAlphaC** [comment]
ty!

**Comment by theirrationalone** [comment]
> Can we use this syntax instead? I don't love the leading underscore

Why not. Ser! ;)

**Pull request merged**